### DISCUSSION TITLE
Migrate to v9?

### DISCUSSION THREAD

**Discussion by Darren120** [gladly-team/next-firebase-auth#310] [General]
When migrate to v9 modular firebase? Thanks.

**Comment by DragoshDX** [comment]
There is a compatibility mode: 
https://firebase.google.com/docs/web/modular-upgrade

In case the maintainers don't want to fully switch perhaps they can implement that.

**Comment by kmjennison** [comment]
This already available on the canary release. See: https://github.com/gladly-team/next-firebase-auth/issues/265

**Reply by Darren120** [reply]
oh ok. ill wait for it to be pushed to main. thanks for ur work.
